src/MOIWrapper.jl

```diff
@@ -82,6 +82,19 @@ function replace_inf(x::Real)
     end
 end
 
+
+function LQOI.get_objectivesense(model::Optimizer)
+    s = get_obj_sense(model.inner)
+    if s == 1.0
+        return MOI.MIN_SENSE
+    elseif s == -1.0
+        return MOI.MAX_SENSE
+    else
+        return MOI.FEASIBILITY_SENSE
```

Comment:
I would add `@assert iszero(s)` before the return just to be sure

Reply:
Thanks @blegat -- just added that.